entries/n/ngpvan.com.json

```
@@ -0,0 +1,10 @@
+{
+  "Site Name": {
+    "domain": "ngpvan.com",
+    "img": "ngpvan.com.png",
+    "tfa": ["sms", "totp"],
+    "url": "https://myaccount.ngpvan.com/Account",
+    "documentation": "https://www.ngpvan.com/actionid/",
+    "categories": ["hosting", "marketing", "other"]
```

The `hosting` category doesn't fit and the `other` category is a last resort. ActionID fits in with services like Action Network, so `marketing` is perfect.

> An ActionID is your gateway to NGP VAN applications, training, support, and feature tutorials.

This `education` category should probably be added. Do you agree, @Carlgo11?

```suggestion
    "categories": [
      "education",
      "marketing"
    ]
```

---

@hkamran80 they're geo-blocking me so I can't even access the main page 😅
You're on your own here @hkamran80!

---

Wow. Here's the relevant paragraph:
> An ActionID is your gateway to NGP VAN applications, training, support, and feature tutorials. Volunteers and campaign staff members who use tools like VAN, Votebuilder, NGP, and MiniVAn have accounts tied to their ActionID. This helps keep your account secure and your profile standardized across our ecosystem. There is no cost to set up an ActionID, and registration takes just a few moments.

---

They do technically provide hosting. From [the Product -> Website page](https://www.ngpvan.com/feature/website/):

> Gorgeous websites that won't break the bank.
>
> Beautiful right out of the box or customized to meet your design specifications, and with an easy-to-use backend that's perfect for campaigns of all sizes; NGP VAN gives you everything you need to create a stunning, secure, and affordable website.
>
> ...
>
> Our award-winning Digital Services team can build a completely custom website that fits your needs, or use our Accelerator tool for an out-of-the-box website. … Create, manage, and organize all your website content (no HTML skills required!) with an intuitive yet powerful interface

`education` feels like a bit of a stretch to me, but I don't feel strongly. Let me know what I should do.

---

Maybe just marketing?

---

That sounds good to me.